engine/src/main/java/com/redhat/cloud/notifications/templates/EmailTemplateFactory.java

```diff
@@ -18,7 +18,12 @@
     private static final String OPENSHIFT = "openshift";
     private static final String CONSOLE = "console";
     private static final String NOTIFICATIONS = "notifications";
+<<<<<<< HEAD
     private static final String RBAC = "rbac";
+=======
+    private static final String SOURCES = "sources";
+
+>>>>>>> df08e47 (feature(Source): Add availabilityStatus event type bindings and email template)
```

Comment:
Looks like there was a conflict.
```suggestion
    private static final String RBAC = "rbac";
    private static final String SOURCES = "sources";
```

Reply:
Thanks @josejulio I had to overlooked it 👍